examples/asteroids.rs

```diff
@@ -56,21 +56,7 @@ async fn main() {
     let mut gameover = false;
 
     let mut screen_center = Vec2::new(screen_width() / 2., screen_height() / 2.);
-    for _ in 0..10 {
```

Comment:
will it remove those beautiful starting asteroids?

Reply:
They are not needed, because a new set is generated every time you start the game. They are beautiful lost references of starting asteroids.

Reply:
Got it!